cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js

```diff
@@ -0,0 +1,148 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import NewOrder from '../../../support/fragments/orders/newOrder';
+import Orders from '../../../support/fragments/orders/orders';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import Organizations from '../../../support/fragments/organizations/organizations';
+import NewOrganization from '../../../support/fragments/organizations/newOrganization';
+import OrderLines from '../../../support/fragments/orders/orderLines';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import Locations from '../../../support/fragments/settings/tenant/location-setup/locations';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import {
+  MATERIAL_TYPE_NAMES,
+  ORDER_FORMAT_NAMES,
+  ACQUISITION_METHOD_NAMES_IN_PROFILE,
+} from '../../../support/constants';
+import Users from '../../../support/fragments/users/users';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventoryHoldings from '../../../support/fragments/inventory/holdings/inventoryHoldings';
+
+describe('Orders: Inventory interaction', () => {
+  const testData = {
+    organization: NewOrganization.getDefaultOrganization({ accounts: 1 }),
+    servicePoint: ServicePoints.defaultServicePoint,
+    instance: {},
+    location: {},
+    orderNumber: '',
+    user: {},
+  };
+
+  before('Create test data', () => {
+    cy.getAdminToken()
+      .then(() => {
+        InventorySearchAndFilter.createInstanceViaApi().then(({ instanceData }) => {
+          testData.instance = instanceData;
+
+          Organizations.createOrganizationViaApi(testData.organization);
+        });
+      })
+      .then(() => {
+        ServicePoints.createViaApi(testData.servicePoint).then(() => {
+          testData.location = Locations.getDefaultLocation({
+            servicePointId: testData.servicePoint.id,
+          });
+
+          Locations.createViaApi(testData.location).then(() => {
+            Orders.createOrderViaApi(NewOrder.getDefaultOrder(testData.organization.id)).then(
+              (number) => {
+                testData.orderNumber = number;
+
+                InventoryHoldings.getHoldingsFolioSource().then((folioSource) => {
+                  InventoryInstances.createHoldingViaAPI({
+                    instanceId: testData.instance.instanceId,
+                    permanentLocationId: testData.location.id,
+                    sourceId: folioSource.id,
+                  }).then(({ id: holdingId }) => {
+                    testData.instance.holdingId = holdingId;
+                  });
+                });
+              },
+            );
+          });
+        });
+      });
+
+    cy.createTempUser([
+      // extra perms
```

Comment:
What is 'extra perms' what does it mean 'perms from test'?

Reply:
and why you're log in by admin?

Reply:
Fixed